src/community/design-system-day-2024/session-information.md

```diff
@@ -63,10 +72,15 @@ When developing a digital service, it's rare to think about how it actually perf
 
 To demonstrate why you should add a performance resource to your Design System, I'll explain how the Scottish Government is tracking captures, user interactions and real questions we've answered.
 
-### Light and dark: product managers in design systems
+### Dual Narrative: Design systems in small versus large organisations
+
+#### Prioritised Pixel Pushing: The case for product within design systems
+
+Michael Palmer, BBC
+
+####
```

Comment:
Tests are failing because of this empty h4. You should be fine if you get rid of this and clean up the spacing.